install/configure_integralstor.sh

```diff
@@ -5,3 +5,4 @@ hardware_vendor=""
 
 echo "/opt/integralstor/integralstor/install/scripts/configure_os.sh $hardware_vendor" | /bin/bash
 echo "/opt/integralstor/integralstor/install/scripts/configure_services.sh" | /bin/bash
+echo "/opt/integralstor/integralstor/install/scripts/configure_applications.sh" | /bin/bash
```

Comment:
Can we change the name of this script to configure_default_applications and then change that script to a python script?

Reply:
Okay, I'll change the script to python.

Reply:
I've made changes to the script and just pushed a commit, please review it.